src/components/nodes/textAnnotation/textAnnotation.vue

```diff
@@ -31,13 +31,13 @@ export default {
       return this.shape;
     },
     updateShape() {
-      let bounds = this.node.diagram.bounds;
       const { height } = this.shape.findView(this.paper).getBBox();
+      const refPoints = `25 ${height} 3 ${height} 3 3 25 3`;
+      let bounds = this.node.diagram.bounds;
+      let textAnnotationLength = this.node.definition.get('text').length;
```

Comment:
Since you're not re-assigning these variables, it's safer to use `const` instead of `let`. I'll eventually add this to our ESLint rules.